packages/integrations/cloudflare/src/index.ts

```diff
@@ -373,39 +386,63 @@ export default function createIntegration(args?: Options): AstroIntegration {
 
 					staticPathList.push(...routes.filter((r) => r.type === 'redirect').map((r) => r.route));
 
+					const strategy = args?.routes?.strategy ?? 'auto';
+
 					// In order to product the shortest list of patterns, we first try to
 					// include all function endpoints, and then exclude all static paths
```

Comment:
I believe this comment is no longer valid, since the logic is changed

Reply:
I have updated the comment.